modules/smtp/site.js

```diff
@@ -226,5 +244,9 @@ $(function() {
             toggle_recip_flds();
         }
         $('.delete_attachment').on("click", function() { return delete_attachment($(this).data('id'), this); });
-    }
+        if (get_url_parameter('reply') === '1'){
```

Comment:
I think get_url_paramater is a bit much for this simple purpose, also we should check for both reply and reply_all. 

Something simple like this should work fine:

```
if (window.location.href.search('&reply=1') !== -1 || window.location.href.search('&reply_all=1') !== -1) {
            replace_cursor_positon ($('textarea[name="compose_body"]'));
}
```